lib/glific/third_party/sheets/sheet.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Glific.Sheets.Sheet do
+  @moduledoc """
+  The minimal wrapper for the base Sheet structure
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias __MODULE__
+
+  alias Glific.{
+    Partners.Organization
+  }
+
+  @required_fields [
+    :label,
+    :url,
+    :organization_id
+  ]
+
+  @optional_fields [:synced_at]
+
+  @type t() :: %__MODULE__{
+          __meta__: Ecto.Schema.Metadata.t(),
+          id: non_neg_integer | nil,
+          label: String.t() | nil,
+          url: String.t() | nil,
+          synced_at: :utc_datetime | nil,
+          organization_id: non_neg_integer | nil,
+          organization: Organization.t() | Ecto.Association.NotLoaded.t() | nil,
+          inserted_at: :utc_datetime | nil,
+          updated_at: :utc_datetime | nil
```

Comment:
Let's add one more column called is_active,
It will help to archive the program and disable the sync process for some time if an org is needed.

Reply:
@pankaj-ag makes sense. made requested changes